resources/js/components/WelcomePage/WelcomeComponent.js

```diff
@@ -1,18 +1,27 @@
 import React from "react";
 import { Link } from "react-router-dom";
-const WelcomeComponent = () => (
-    <div>
-        <h2 className="heading">WELCOME TO JOB PORTAL</h2>
-        <div className="button-pos">
-            <Link to="/postJob">
-                <button className="welcome-button text-white btn-primary">
-                    ADD Job
-                </button>
-            </Link>
+import ButtonComponent from "../Button/ButtonComponent";
+const button = {
```

Comment:
button is a HTML keyword . can you please change it to buttonObj

Reply:
last one plz

Reply:
Done!!